easybuild/easyconfigs/r/Ruby/Ruby-2.1.5.eb

```diff
@@ -0,0 +1,52 @@
+#    With <3 for EasyBuild
+#    
+#    EasyConfig for Ruby 2.1:
+#    ----------------------------------------------------------------------------
+#    Copyright: 2014 - Gregor Mendel Institute of Molecular Plant Biology GmBH
+#    License: MIT
+#    Authors: Aaron <aaron.zauner@gmi.oeaw.ac.at> Zauner
+#    ----------------------------------------------------------------------------
+
+name = 'Ruby'
+version = '2.1.5'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://cache.ruby-lang.org/pub/ruby/']
+
+sanity_check_paths = {
+    'files': ['bin/ruby', 'bin/rake', 'bin/gem', 'bin/testrb', 'bin/erb',
+              'bin/ri', 'bin/irb', 'bin/rdoc', 'lib/libruby.so.2.1.0',
+              'lib/libruby-static.a', 'lib/libruby.so'],
+    'dirs': ['lib/ruby/2.1.0', 'lib/ruby/gems', 'lib/pkgconfig', 'include/ruby-2.1.0']
+}
+
+configopts = "--disable-install-doc --enable-shared"
+
+moduleclass = 'lang'
+
+name_tmpl = '%(name)s-%(version)s.gem'
+
+ext_options = {
+    'source_urls': [
+        'http://rubygems.org/downloads/',  # gem archive
+    ],
+    'source_tmpl': name_tmpl,
+}
+
+exts_list = [
+    ('ffi',       '1.9.6', ext_options),
+    ('childprocess', '0.5.5', ext_options),
+    ('json',      '1.8.2', ext_options),
+    ('cabin',     '0.7.1', ext_options),
+    ('backports', '3.6.4', ext_options),
+    ('arr-pm',    '0.0.9', ext_options),
+    ('clamp',     '0.6.4', ext_options),
+    ('fpm',       '1.3.3', ext_options),
```

Comment:
not sure what you mean. Do you think we should drop the foss one for this (or at least remove the packaging modules?)